services/application.js

```diff
@@ -410,6 +413,43 @@ class Application {
             url: this.emailParams.url
         })
     }
+
```

Comment:
The whole temporary credential functionality shouldn't be added to application.js. It should be in its own file. Because it's no related to "Application" aka. "Submission Request" or "Questionnaire".